src/app/components/MediaLoader/index.test.tsx

```diff
@@ -191,19 +195,31 @@ describe('MediaLoader', () => {
       await act(async () => {
         render(
           <MediaPlayer
-            blocks={onDemandTvBlocks as MediaBlock[]}
-            pageIdentifierOverride="hindi.bbc_hindi_tv.tv.w172zm8b4tlpzxh.page"
+            blocks={onDemandTvBlocksWithOverrides as MediaBlock[]}
             embedded
           />,
           { service: 'hindi' },
         );
       });
 
-      expect(buildConfigSpy).toHaveBeenCalledWith(
+      expect(buildConfigSpy.mock.calls[0][0]).toEqual(
         expect.objectContaining({
           counterName: 'hindi.bbc_hindi_tv.tv.w172zm8b4tlpzxh.page',
         }),
       );
+      expect(buildConfigSpy.mock.calls[0][0].blocks).toEqual(
```

Comment:
Is this expectation necessary, as it's asserting on the fixture data which is passed to the MediaPlayer...?

Reply:
I figured that since we are passing a modified block tot he MediaLoader then
we ought to assert for it in the same way we are asserting for `counterName`.
Happy to remove it if deemed unnecessary.

Reply:
We can leave it for now, I don't want to hold up this work.